src/vos/evtree.c

```diff
@@ -2598,32 +2590,39 @@ evt_sdist_split(struct evt_context *tcx, bool leaf, struct evt_node *nd_src,
 {
 	struct evt_node_entry	*entry_src;
 	struct evt_node_entry	*entry_dst;
+	struct evt_rect		*mbr;
 	int			 nr;
 	int			 delta;
 	int			 boundary;
 	bool			 cond;
 	int64_t			 dist;
 
+	mbr = evt_node_mbr_get(tcx, nd_src);
+
 	D_ASSERT(nd_src->tn_nr == tcx->tc_order);
 	nr = nd_src->tn_nr / 2;
 
 	nr += nd_src->tn_nr % 2;
 
-	entry_src = evt_node_entry_at(tcx, nd_src, nr - 1);
-	dist = evt_mbr_dist(evt_node_mbr_get(tcx, nd_src), &entry_src->ne_rect);
+	entry_src = evt_node_entry_at(tcx, nd_src, nr);
```

Comment:
Note that using nr is safe here because this is only called on split (full node) and the minimum order is 4.   So tn_nr / 2 + 1 will always be < order.   The + 1 only happens for odd numbers so worst case is order of 5 in which case nr would be 3.